internal/pkg/cli/env_init.go

```diff
@@ -672,6 +672,13 @@ func (o *initEnvOpts) adjustVPCConfig() *config.AdjustVPC {
 	}
 }
 
+func (o *initEnvOpts) setAllowVPCIngress() *bool {
+	if o.allowVPCIngress {
+		return aws.Bool(o.allowVPCIngress)
```

Comment:
I feel like this is a bit strange 🤔 If i'm following this the same as setting:
```suggestion
		return aws.Bool(true)
```
I agree with Wanxian's feedback below, I can't tell what the pointer is allowing us to do instead of just a `bool`